heron/ui/src/python/handlers/common/__init__.py

```diff
@@ -1,4 +1,5 @@
+''' common module '''
 __all__ = ['common']
 
 from heron.ui.src.python.handlers.common.consts import *
-from heron.ui.src.python.handlers.common.utils  import *
+from heron.ui.src.python.handlers.common.utils import *
```

Comment:
On line 151 of file handlers/api/topology.py, `diameter = common.graph.TopologyDAG(lplan).diameter()`, `graph` is also used as member of the common module. We need to add one line here to include it in the common module `from heron.ui.src.python.handlers.common.graph import *`.